tests/eclient/testdata/ctrl_cert_change.txt

```diff
@@ -0,0 +1,70 @@
+# Test of controller certificate change
+
+{{$port := "2223"}}
+
+{{$userdata := "variable=value"}}
+{{define "eclient_image"}}docker://{{EdenConfig "eden.eclient.image"}}:{{EdenConfig "eden.eclient.tag"}}{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:chmod] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+eden network create 10.11.12.0/24 -n n1
+eden pod deploy -n eclient1 --memory=512MB --networks=n1 {{template "eclient_image"}} -p {{$port}}:22 --metadata={{$userdata}}
+
+test eden.app.test -test.v -timewait 20m RUNNING eclient1
+
+# generate new controller certificate
+eden utils gen-signing-cert -o {{EdenConfig "eden.tests"}}/eclient/image/cert/signing-new.pem
+
+# upload new certificate to controller, resign old config and reapply it
+eden adam upload-cert --cert-file {{EdenConfig "eden.tests"}}/eclient/image/cert/signing-new.pem
+
+# wait for changes to be applied
+test eden.lim.test -test.v -timewait 15m -test.run TestLog -app eclient1 -out content 'content:Rebuilding.intended.global.config,.reasons:.reconnecting.app'
```

Comment:
Can you please clarify from where we expect the log? Are they aligned with latest changes of https://github.com/lf-edge/eden/blob/master/tests/eclient/image/files/entrypoint.sh?

Reply:
We expect this log message from [here](https://github.com/lf-edge/eve/blob/e5db3e506dc39135037d82d23c40785e069310cd/pkg/pillar/nireconciler/linux_reconciler.go#L400).

Sorry, I don't see how it relates to the file that you mentioned... But the branch has the newest version of `entrypoint.sh`

Reply:
Ah. I see, it is log from EVE-OS (and you use `TestLog`). Seems `-app eclient1` not used [here](https://github.com/lf-edge/eden/blob/c2ba0752c0587dae3f5dd5d2fa74f56f6fcf2e8c/tests/lim/lim_test.go#L154-L193) and confuse me.

Reply:
You are right, don't know where I got it from. Removed.